markdown/bitburner.stanek.canplacefragment.md

```diff
@@ -18,7 +18,7 @@ canPlaceFragment(rootX: number, rootY: number, rotation: number, fragmentId: num
 |  --- | --- | --- |
 |  rootX | number | rootX Root X against which to align the top left of the fragment. |
 |  rootY | number | rootY Root Y against which to align the top left of the fragment. |
-|  rotation | number | rotation A number from 0 to 3, the mount of 90 degree turn to take. |
+|  rotation | number | rotation A number from 0 to 3, the amount of 90-degree turns to take. |
```

Comment:
This method signature (at least) is duplicating param names that should be trimmed.

![image](https://github.com/bitburner-official/bitburner-src/assets/141260118/2b6a9404-4237-43e7-842c-a90ac07ee8b3)

Reply:
I'm not sure I understand what you are saying here. Can you clarify?

Reply:
I think they're talking about how stuff like "rootX", "rotation" etc. is duplicated needlessly. It should be specified as the argument to param, but doesn't need to be duplicated in the description.